# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/avm/tuxnes/gcc34.patch
# ROCK Linux is Copyright (C) 1998 - 2006 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./emu.c.orig	2004-09-13 02:50:10.642433736 +0200
+++ ./emu.c	2004-09-13 02:52:00.656709024 +0200
@@ -890,7 +890,7 @@
       len = strlen(palfile) + 1;
       if (! (buffer = malloc(len)))
 	{
-	  perror (__FUNCTION__ ": malloc");
+	  perror ("loadpal 893: malloc");
 	  return;
 	}
       memcpy (buffer, palfile, len);
@@ -912,7 +912,7 @@
       len = strlen(filename) + 1;
       if (! (buffer = malloc(len)))
 	{
-	  perror (__FUNCTION__ ": malloc");
+	  perror ("loadpal 915: malloc");
 	  return;
 	}
       memcpy (buffer, filename, len);
@@ -924,7 +924,7 @@
 	return;
       if (!(palfile = malloc ((len = strlen (filename)) + 11)))
         {
-          perror (__FUNCTION__ ": malloc");
+          perror ("loadpal 927 : malloc");
           return;
         }
       strcpy (palfile, filename);
@@ -1602,7 +1602,7 @@
 
     if (! (basefilename = malloc(baseend - basestart + 1)))
       {
-        perror (__FUNCTION__ ": malloc");
+        perror ("main 1605: malloc");
         exit (1);
       }
 
